src/player/ui.rs

```diff
@@ -88,7 +88,7 @@ async fn input(sender: Sender<Messages>) -> eyre::Result<()> {
                 event::MediaKeyCode::Play => Messages::PlayPause,
                 event::MediaKeyCode::Pause => Messages::PlayPause,
```

Comment:
Intentionally didn't change those two. I have a media key on my keyboard that looks like "play pause" but it's a lie... `wev` tells me it's `XF86AudioPlay` instead. I would have to use `stop` key on my keyboard to pause and it's little awkward. I would propose to keep it as is until someone complains (if ever).